Library/OcCpuLib/OcCpuLib.c

```diff
@@ -786,7 +815,12 @@ ScanAmdProcessor (
       if (MaxBusRatio == 0) {
         Cpu->FSBFrequency = 100000000; // 100 MHz like Intel part.
       } else {
```

Comment:
better to change to else if, and not introduce a new level of indentation.

Reply:
Updated. Hopefully that fixes your concerns